src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
+
+  min-width: 0;
+  padding: 0;
+
+  width: 40px;
+  height: 40px;
+
+  line-height: 24px;
+  border-radius: 50%;
```

Comment:
Because if you have a 24x24 square, then 50% of all sides will be used to archive a responsive perfect circle.

So it's needed to make the element round, like in polymer-icon-button and Material 1